src/modules/bpmn-diff-view/bpmn-diff-view.ts

```diff
@@ -429,10 +429,10 @@ export class BpmnDiffView {
   }
 
   private _updateDiffView(): void {
-    if (this.currentDiffMode === DiffMode.CurrentVsPrevious) {
+    if (this.currentDiffMode === DiffMode.NewVsOld) {
```

Comment:
When editing this you could also extract the business rules:

```suggestion
	const diffModeIsNewVsOld: boolean = this.currentDiffMode === DiffMode.NewVsOld;
	const diffModeIsOldVsNew: boolean = this.currentDiffMode === DiffMode.NewVsOld;
	
    if (diffModeIsNewVsOld) {
```